### PR TITLE
refactor: simplify cljs macro requires

### PR DESCRIPTION
**Pull request by opqdonut**
Requiring macros directly (without `:require-macros` etc) tends to just work in cljs these days.

### CODE REVIEW

**Review by marharyta — APPROVED** [review]

File: src/clj/rems/guide_utils.clj [hunk 1]
```diff
@@ -1,7 +1,6 @@
-(ns rems.guide-macros
-  "Utilities for component guide."
+(ns rems.guide-utils
```
Comment (Macroz) [inline src/clj/rems/guide_utils.clj:1]: Why plural? The others are `-util`?

File: src/clj/rems/guide_utils.clj [hunk 2]
```diff
@@ -11,21 +10,18 @@
...
 (defmacro component-info [component]
   ;; TODO we'd like to use normalize-file-path on the component
   ;; metadata at compile time, just like we do for namespaces above.
   ;; However it seems that there's no way to get the metadata at
   ;; compile time (e.g. resolve and ns-resolve return nil). Thus we
-  ;; normalize the path at runtime in rems.guide-functions/link-to-source.
+  ;; normalize the path at runtime in link-to-source.
   `(let [m# (meta (var ~component))]
-     (rems.guide-functions/render-component-info
+     (render-component-info
```
Comment (Macroz) [inline src/clj/rems/guide_utils.clj:24]: Hmm so this works these days?

**Review by Macroz — APPROVED** [review]
Slightly baffling but if this is the modern way let's go for it!

I think I would rename from plural to singular to match the others.

File: src/clj/rems/guide_utils.clj (hunk 1 above)
Comment (opqdonut) [reply src/clj/rems/guide_utils.clj:1]: Yeah that was a leftover from the previous `guide-utils` namespace. I'll fix it!

File: src/clj/rems/guide_utils.clj (hunk 2 above)
Comment (opqdonut) [reply src/clj/rems/guide_utils.clj:24]: It's just normal macro behaviour where the vars get qualified with the current namespace name, which happens to be the name of the corresponding cljs namespace where these functions are defined...

Comment (Macroz) [reply src/clj/rems/guide_utils.clj:24]: Yeah, you moved these into the same ns though not to the same file.

**Pull request merged**